src/prpy/planning/ompl.py

```diff
@@ -245,3 +246,100 @@ def ShortcutPath(self, robot, path, timeout=1., **kwargs):
         }, append=True)
 
         return output_path
+
+class ConstrainedOMPLPlanner(OMPLPlanner):
+    def __init__(self):
+        OMPLPlanner.__init__(self, algorithm='CRRTConnect')
+
+    @ClonedPlanningMethod
+    def PlanToTSR(self, robot, tsrchains, planner_range=0.1, ompl_args=None, **kw_args):
+        """
+        Plan using the given TSR chains with OMPL. 
+        @param robot
+        @param tsrchains A list of TSRChain objects to respect during planning
+        @param planner_range the max stepsize for the planner before a projection
+         is applied
+        @param ompl_args ompl RRTConnect specific parameters
+        @return traj
+        """
+        if ompl_args is None:
+            ompl_args = {}
+        ompl_args['range'] = planner_range
```

Comment:
From what I remember (and I'll have to take a closer look to determine this more closely), the goal here was to emulate the way CBiRRT places vertices in its RRT after each projection.  Since the default `range` parameter is rather large, we currently overwrite it only for constrained planners.